content/en/docs/Installation/trustedboot.md

```diff
@@ -121,6 +121,44 @@ title My Awesome OS
 Remember when you build upgrade mediums, to also add your branding, otherwise the new active image will be named differently than your fallback and recovery ones.
 {{% /alert %}}
 
+### Additional efi entries
+
+When booting in UKI mode, the cmdline is part of the signed artifact. This means that in order to pass additional options through the cmdline,
+an additional efi entry is needed. `enki build-uki` supports the following arguments:
```

Comment:
I don't think "an additional efi entry is needed" in all cases, like the extend one

Reply:
Good point. Re-phrasing.